src/components/row/row.style.js

```diff
@@ -4,6 +4,7 @@ import baseTheme from "../../style/themes/base";
 import StyledColumn from "./column/column.style";
 
 const sizes = {
+  none: 0,
```

Comment:
none should be added to the MDX gutter example to show what none looks like, the test story knobs and to the row component so it shows it as an option in the docs.